pkg/chartvalues/apiextensions_app_e2e_test.go

```diff
@@ -131,6 +135,54 @@ secrets:
     test: "secret"`,
 			errorMatcher: nil,
 		},
+		{
+			name:   "case 2: no configmap and secret",
+			config: newAPIExtensionsAppE2EConfigFromFilled(removeConfigMapAndSecret),
+			expectedValues: `
+apps:
+  - name: "test-app"
+    namespace: "default"
+    catalog: "test-app-catalog"
+    kubeConfig:
+      inCluster: false
+      secret:
+        name: "test-kubeconfig-secret"
+        namespace: "default"
+    version: "1.0.0"
+  # Added chart-operator app CR for e2e testing purpose.
+  - name: "chart-operator"
+    namespace: "giantswarm"
+    catalog: "giantswarm-catalog"
+    kubeconfig:
+      inCluster: "true"
+    version: "0.9.0"
+
+appCatalogs:
+  - name: "test-app-catalog"
+    title: "test-app-catalog"
+    description: "giantswarm app catalog"
+    logoURL: "http://giantswarm.logo.catalog.png"
+    storage:
+      type: "helm"
+      url: "https://giantswarm.github.com/sample-catalog"
+  - name: "giantswarm-catalog"
+    title: "giantswarm-catalog"
+    description: "giantswarm catalog"
+    logoUrl: "http://giantswarm.com/catalog-logo.png"
+    storage:
+      type: "helm"
+      url: "https://giantswarm.github.com/giantswarm-catalog/"
+
+appOperator:
+  version: "1.0.0"
+
+
```

Comment:
Those are ugly white space, but can't find a way to formatting well with go templates :(

Reply:
Yes these are hard to manage. Could we have a constant so the UT code is cleaner?

Reply:
Sure, I move all of the expected values as const.